src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1096,5 +1102,41 @@ private static void ThrowMinMaxException<T>(T min, T max)
         {
             throw new ArgumentException(SR.Format(SR.Argument_MinMaxValue, min, max));
         }
+
+        public static double ScaleB(double x, int n)
```

Comment:
Was this ported from an existing implementation?

Based on https://github.com/dotnet/runtime/issues/40956#issuecomment-693022684, it sounds like its based on the MUSL implementation which is MIT (https://git.musl-libc.org/cgit/musl/tree/COPYRIGHT) and so we need a comment around here and also need to update the https://github.com/dotnet/runtime/blob/master/THIRD-PARTY-NOTICES.TXT.

CC. @richlander on if there is anything else required.

Reply:
The MUSL implementation came from Sun's implementation, so we may need the notice for that too (or just have notice for the Sun's implementation if it was not changed for MUSL).

Reply:
Yes, this was based on the musl implementation. I added links to the source for each function and included both the musl and Sun license. Let me know if I missed or can add anything.